src/Couchbase.TravelSample/Program.cs

```diff
@@ -1,33 +1,80 @@
-using Microsoft.Extensions.Options;
-
+using Couchbase.Core.Exceptions.KeyValue;
 using Couchbase.Extensions.DependencyInjection;
+using Couchbase.KeyValue;
 using Couchbase.TravelSample.Models;
+using Microsoft.OpenApi.Models;
+using FluentValidation;
+
+using Route = Couchbase.TravelSample.Models.Route;
 
 var builder = WebApplication.CreateBuilder(args);
+const string devSpecificOriginsName = "_devAllowSpecificOrigins";
+
+//global pointer to inventory scope
+IScope? inventoryScope = null;
 
-/// <summary>
-/// dev origins used to fix CORS for local dev/qa debugging of site
-/// </summary>
-const string _devSpecificOriginsName = "_devAllowSpecificOrigins";
+const string airportCollection = "airport";
+const string airlineCollection = "airline";
+const string routeCollection = "route";
 
 // Add services to the container.
 // Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
 builder.Services.AddEndpointsApiExplorer();
-builder.Services.AddSwaggerGen();
+builder.Services.AddSwaggerGen(options =>
+{
+    options.SwaggerDoc("v1", new OpenApiInfo
+    {
+        Version = "1.0",
+        Title = "Quickstart in Couchbase with C# and ASP.NET Minimal API",
+        Description = "A quickstart API using C# and ASP.NET with Couchbase and travel-sample data. \n\n"
```

Comment:
this is NOT the proper way to handle strings in .NET.  Use a String Builder or string interpolation.

Reply:
Missed it but done now.